var/spack/repos/builtin/packages/http-ping/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class HttpPing(MakefilePackage):
+    """http_ping is like the regular ping command, except that it sends
+    HTTP requests instead of ICMP echo requests."""
+
+    homepage = "http://www.acme.com/software/http_ping/"
+    url      = "http://www.acme.com/software/http_ping/http_ping_09Mar2016.tar.gz"
+
+    version('09Mar2016', sha256='6bdc570c776a760d2c08f7d18e00b0edd74cf603400929c66c512801b6bb5871')
```

Comment:
This will be problematic if anyone ever adds another version, as the version won't sort correctly. I would change the version of `2016-03-09` and either manually specify the URL for each version or use Python's datetime module to convert from one format to another in `url_for_version`. This is a suggestion, not a requirement, it could wait until someone actually needs another version.